react/next/index.tsx

```diff
@@ -199,27 +211,43 @@ const SliderNext: FC<SliderProps> = props => {
           props.removeArrowOnDeviceType.indexOf(state.deviceType) > -1))
     )
 
+  const hasThumbsleft = props.thumbnails && props.thumbnails.position === 'left'
+
+  const containerClasses = `${props.classNames!.sliderContainer} flex w-100`
+
+  const sliderContainerClasses = `${
+    props.classNames!.sliderContainer
+  } ${hasThumbsleft && 'order-1'} flex items-center relative overflow-hidden`
+
   return (
     <section
-      className={`${
-        props.classNames!.container
-      } flex items-center relative overflow-hidden`}
-      ref={containerRef}
       role="region"
       aria-roledescription="carousel"
       aria-label={props.label}
+      className={containerClasses}
     >
-      <SliderTrack
-        id={itemsId}
-        className={props.classNames!.slider}
-        transform={state.transform}
-        transition={props.transition!}
+      <div
+        className={sliderContainerClasses}
+        style={{
+          width: props.thumbnails
+            ? `calc(100% - ${props.thumbnails.width})`
+            : `100%`,
+        }}
+        ref={containerRef}
       >
-        <SlideList {...state} {...props} />
-      </SliderTrack>
-      {shouldShowArrows && renderLeftArrow()}
-      {shouldShowArrows && renderRightArrow()}
-      {props.showDots && renderDotsList()}
+        <SliderTrack
+          id={itemsId}
+          className={props.classNames!.slider}
+          transform={state.transform}
+          transition={props.transition!}
+        >
+          <SlideList {...state} {...props} />
+        </SliderTrack>
+        {shouldShowArrows && renderLeftArrow()}
+        {shouldShowArrows && renderRightArrow()}
+        {props.showDots && renderDotsList()}
+      </div>
+      {props.thumbnails && renderTumbnails()}
```

Comment:
I think that using the component directly here instead of calling `renderTumbnails` makes the code more readable. Also, fix the typo if you decide to keep the function

Reply:
I'll fix the typo, and keep using the function. I'll do this to keep consistency with `arrows` and `dots` usage.